cypress/integration/mailinglist-signup-flow.spec.js

```diff
@@ -224,6 +224,41 @@ describe(`Feature - Mailing list sign up : Tests execution date and time : ${new
 			.should("exist")
 			.should("have.text", "You’ve already signed up");
 	});
+
+	it("It shows Privacy policy details to the user if he clicks on link", function () {
+		mailingListSignUp.getFirstName().type(this.testInputData.firstName);
+		mailingListSignUp.getLastName().type(this.testInputData.lastName);
+		let rnum = Math.floor(Math.random() * 1000000000 + 1);
+		let email = `testuser${rnum.toString()}@mail.co.uk`;
+		mailingListSignUp.getEmailAddress().type(email);
+		cy.get("#mailing-list-steps-name-degree-status-id-field").select("Final year");
+		mailingListSignUp.getNextStep().click();
+		mailingListSignUp
+			.getStage()
+			.select(this.testInputData.howCloseAreYoutoApplyingForTeacherTraining);
+		mailingListSignUp.getNextStep().click();
+		mailingListSignUp.getSubjectToTeach().select(this.testInputData.whichSubjectdoYouWantToTeach);
+		mailingListSignUp.getNextStep().click();
+		mailingListSignUp.getPostcode().type(this.testInputData.postCode);
+		mailingListSignUp.getNextStep().click();
+		cy.contains("privacy policy").click();
+		cy.get('a[href="/privacy-policy?id=8da7ae80-82f2-ea11-a815-000d3a44afcc"]').should(
+			"have.attr",
+			"target",
+			"blank"
+		);
+		cy.visit("/privacy-policy?id=8da7ae80-82f2-ea11-a815-000d3a44afcc", {
+			auth: {
+				username: Cypress.env("HTTPAUTH_USERNAME"),
+				password: Cypress.env("HTTPAUTH_PASSWORD"),
+			},
+		});
+		cy.get(".content__left > h2").should("exist").should("have.text", "Privacy Policy");
```

Comment:
I think these selectors are too brittle, especially `:nth-child(3) > strong` - that one is asking to break 😄

Reply:
Resolved